x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -166,7 +168,7 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
 
     core.application.register({
       id: `${APP_ID}:${SecurityPageName.hosts}`,
-      title: 'Hosts',
+      title: i18n.translate('xpack.securitySolution.hostsPage.title', { defaultMessage: 'Hosts' }),
```

Comment:
Do we already have this String (and other in that you i18n'd here) defined in `/app/home/translations.ts`? If so, I would propose we use those here.